README.md

```diff
@@ -12,14 +10,8 @@ This project is developed based on [Checker Framework](http://types.cs.washingto
 
 - txiang61/checker-framework
 - txiang61/checke-framework-inference
-- jsr308-langtools
-- annotation-tools
```

Comment:
annotation-tools should stay? Maybe change to `opprop/annotation-tools`?

Reply:
Building CF/CFI will automatically download `annotation-tools` and other dependencies. What do you think?